web/modules/custom/farm_page/templates/farm-homepage.html.twig

```diff
@@ -1,8 +1,35 @@
-{% if is_exist == true %}
-{{ store.field_description.value }}
-	{% for image in store_images %}
-	<img src={{image}}>
-	{% endfor %}
+{% if data.is_exist == TRUE %}
```

Comment:
code format.

Reply:
formatted

Reply:
it's not formatted, should be two spaces.